nixfmt.cabal

```diff
@@ -0,0 +1,80 @@
+cabal-version:       2.0
+
+-- © 2022-2024 Serokell <hi@serokell.io>
+-- © 2022-2024 Lars Jellema <lars.jellema@gmail.com>
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+name:                nixfmt
+version:             0.6.0
+synopsis:            An opinionated formatter for Nix
+description:
+  A formatter for Nix that ensures consistent and clear formatting by forgetting
+  all existing formatting during parsing.
+homepage:            https://github.com/serokell/nixfmt
+bug-reports:         https://github.com/serokell/nixfmt/issues
+license:             MPL-2.0
+license-file:        LICENSE
+author:              Lars Jellema
+maintainer:          lars.jellema@gmail.com
+copyright:           2022 Serokell, 2022 Lars Jellema
+category:            Development
+build-type:          Simple
+extra-source-files:  README.md, CHANGELOG.md
+
+library
```

Comment:
Looks weird, because we don't have library files in the repo (and we shouldn't have them).

Reply:
> because we don't have library files in the repo

We indeed don't have them in the repo, but `nixfmt` sources are copied prior to the build, see https://github.com/serokell/nixfmt-website/blob/01391e3/flake.nix#L35

Reply:
While I agree this doesn't look very :proper:, but maybe using `nixfmt` as a Haskell dependency has some issues :thinking:

Reply:
Okay, I think we shouldn't care much about having something :proper: in this repo, but then let's add some comment somewhere in this file, so that people like me don't get confused. And we should probably update things like `homepage`, `description`, since it's easy and quick to do.

Reply:
For some reason, i initially didnt think of extracting nixfmt as a separate package, fixed. I also updated the description, home page, etc.